Resources/Prototypes/Floof/Entities/Mobs/NPCs/xenopet.yml

```diff
@@ -0,0 +1,308 @@
+- type: entity
+  name: basic
+  id: MobSpaceFriend
+  parent: MobXenoRounyNPC
+  description: It looks friendly. Why don't you give it a hug?
+  components:
+  - type: NpcFactionMember
+    factions:
+    - PetsNT
+  - type: MovementAlwaysTouching
+  - type: MeleeWeapon
+    soundHit:
+        path: /Audio/Weapons/Xeno/alien_claw_flesh3.ogg
+    angle: 0
+    animation: WeaponArcClaw
+    damage:
+      types:
+        Blunt: 5
+        Slash: 5
+  - type: Puller
+    needsHands: false
+  - type: GhostTakeoverAvailable

```

Comment:
MobSpaceFriend is only used two, you could just merge those those with MobXenoNeutralRouny

and parent MobXenoFriendShaped to MobXenoNeutralRouny

Reply:
I found this easier as it was less code to just specify the difference in faction than it was to specify a different sprite and handle all the differences (combat values and such) created by having it parented to something that's own parent is in fact the MobXenoRounyNPC